bibliopixel/animation/animation.py

```diff
@@ -119,7 +119,8 @@ def run_context(self):
 
         self.check_delay()
 
-        self.preRun(self.runner.amt)
+        self.layout.all_off()
```

Comment:
"if you need a specific starting state"

Exactly the reverse of what I wanted.  

No, I meant that an animation might _not_ have a specific starting state at all, but might want to use "whatever was already in the buffer from what was running".

In particular, to do "professional" quality animations you need a way to guarantee smooth transitions.

Having a mechanism that guarantees a smooth transition from an unknown animation that has just completed is impossible if that buffer is pre-cleared for you.